src/SeleniumLibrary/utils/webdrivercache.py

```diff
@@ -17,33 +17,34 @@
 from robot.utils import ConnectionCache
 
 
-class BrowserCache(ConnectionCache):
+class WebDriverCache(ConnectionCache):
 
     def __init__(self):
-        ConnectionCache.__init__(self, no_current_msg='No current browser')
+        ConnectionCache.__init__(self, no_current_msg='No current driver')
```

Comment:
This should still say `browser`. And have `.` at the end.

If that message is ever used, it's used by `ConnectionCache` as a message for `RuntimeError`. We have our own `NoOpenBrowser` now, but there's no good API to use that here. We could set `self._no_current` to an object using our exception (and it could extend `NoConnection`) but I'm not sure is that worth the effort.

Reply:
I did change to driver -> browser. 

If the start seeing the `RuntimeError`in real life, lets then decide what we should do.